test/fixtures/mocks/res.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash'),
```

Comment:
Very minor but looks like you are only using `_.constant` in this file, do you mind requesting just that method from lodash.